source/frontend/src/components/measurements/VolumeView.tsx

```diff
@@ -0,0 +1,42 @@
+import { VolumeUnitTypes } from 'constants/volumeUnitTypes';
+import { Col, Row } from 'react-bootstrap';
+import { convertVolume, formatNumber } from 'utils';
+
+import { StyledGreenBlue } from './styles';
+
+interface IVolumeViewProps {
+  volume?: number;
+  unitCode?: string;
+}
+
+const VolumeView: React.FunctionComponent<IVolumeViewProps> = ({ volume, unitCode }) => {
+  const meters = convertVolume(volume || 0, unitCode || '', VolumeUnitTypes.CubicMeters);
+  const feets = convertVolume(volume || 0, unitCode || '', VolumeUnitTypes.CubicFeet);
+
+  return (
+    <Row>
+      <Col>
+        <StyledGreenBlue>
+          <Row>
+            <Col className="text-right">{formatNumber(meters)}</Col>
+            <Col>
+              <span>
+                metres<sup>3</sup>
+              </span>
+            </Col>
+          </Row>
+          <Row>
+            <Col className="text-right">{formatNumber(feets)}</Col>
```

Comment:
nit: feets vs feet. the singular of feet is foot, the plural is feet. ie: 1 foot 3 feet